### PR TITLE
SCT version moved to 315e3a56a6d9261d4fad4c1950f2d01a052eeba4

### PR DESCRIPTION
**Pull request by Guru9902**

### CODE REVIEW

File: common/config/sr_es_common_config.cfg [hunk 1]
```diff
@@ -34,7 +34,7 @@ LINUX_KERNEL_VERSION=6.4
 EDK2_SRC_VERSION=edk2-stable202402
 
 # EDK2-TEST source tag from  https://github.com/tianocore/edk2-test
-SCT_SRC_TAG=032822757792c5d4d0bfed1fd8524e69ef4f2d17
+SCT_SRC_TAG=315e3a56a6d9261d4fad4c1950f2d01a052eeba4
```
Comment (edhay) [inline common/config/sr_es_common_config.cfg:37]: Why is this moved to a older commit of edk2-test?

Comment (chetan-rathore) [reply common/config/sr_es_common_config.cfg:37]: SCT version movement of systemready image, has dependency on bbr-acs edk2-test patches.
The patch of 032822757792c5d4d0bfed1fd8524e69ef4f2d17 is under development, SR build is passing in github actions due to no exit condition if patch apply fails.
Ex
https://github.com/ARM-software/bbr-acs/blob/8f6065e877f70938d5caa53f96e57a88f0782bc1/common/scripts/build-sct.sh#L173

The SCT version will be changes along with bbr-acs patch PR.
